suricata.yaml.in

```diff
@@ -449,6 +456,10 @@ outputs:
       # remain open for filestore by Suricata. Default value is 0 which
       # means files get closed after each write
       #max-open-files: 1000
+      # uncomment to write to tmp files with the given prefix.
+      # No effect if write-meta is not yes.
+      #working-file-prefix: .
```

Comment:
Why does this not take affect if writing metadata is not enabled?

What would be the downside of removed the option of "working-file-prefix" and just make it the default all the time. Its a much better standard practice.

Reply:
Renaming the file to remove the prefix happens when LogFilestoreLogCloseMetaFile is called. I guess this could still work if instead of returning when !FileWriteMeta(), we could enclose the entire function up to renaming the file in an if block.

What do you think is better?

Reply:
I don't see a problem with the prefix always being "."; I just wasn't sure if it would be useful to be configurable or not.

Reply:
I'm wondering if the dot prefix makes sense as it makes it a hidden file. Seems that the browsers I use settled on '.part' suffix instead.

Wrt the renaming, I think it should be done in such a way that it does not depend on the metadata functionality being enabled.

Reply:
I like the use of a suffix to note that file is still a partial (or live, active whatever).

Reply:
Ok, that makes sense. Do you think the suffix should be configurable? My default would be ".tmp", but I'm not sure whether everyone would agree.

Reply:
I'm OK with .tmp.

Reply:
Updated version in #3008.